Week1/homework/js2-exercises/5-catWalking.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Cat Walk</title>
+  </head>
+  <body>
+    <img style="position:absolute;" src="http://www.anniemation.com/clip_art/images/cat-walk.gif" />
+    <script >
+        const img = document.querySelector('img');
+        const srcCatWalking = 'http://www.anniemation.com/clip_art/images/cat-walk.gif';
+        const srcCatDancIng = 'https://media2.giphy.com/media/3mXcJgVhPxWZa/giphy.gif';
+        let left = 0
+
+        function catWalk() {
+
+            setInterval(frame, 50)
+            function frame() {
```

Comment:
Good work however I wouldn't declare a function inside of a function simply because no one can use the inner function outside the outer function.
I think it would be better to keep all functionality inside of `catWalk` function and then call `catWalk` with setInterval every 50 ms.
```javascript
function catWalk() {
    ... all functionality
}

setInterval(catWalk, 50);
```